### PR TITLE
Problem: adding console.log for tx failure error (fix #345)

### LINKED ISSUE #345: Problem: adding console.log for tx failure error

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution:add debug log

### CODE REVIEW

File: src/service/rpc/NodeRpcService.ts [hunk 1]
```diff
@@ -62,6 +62,8 @@ export class NodeRpcService implements INodeRpcService {
     try {
       const signedBytes = Bytes.fromHexString(signedTxHex).toUint8Array();
       const broadcastResponse = await this.client.broadcastTx(signedBytes);
+      // eslint-disable-next-line no-console
+      console.log(`tx failure= ${JSON.stringify(broadcastResponse)}`);
```
Comment (eddycpc) [inline src/service/rpc/NodeRpcService.ts:66]: I'm wondering if there's a reason would we want to console log in release builds? 

The console tab is being hidden in release builds, so even if it logs we can't see the logs

**Review by eddycpc — CHANGES_REQUESTED** [review]
Left a comment

File: src/service/rpc/NodeRpcService.ts (hunk 1 above)
Comment (leejw51crypto) [reply src/service/rpc/NodeRpcService.ts:66]: ok, i see

**Comment by leejw51crypto** [comment]
replace this issue with this one
https://github.com/crypto-com/chain-desktop-wallet/issues/349

**Pull request closed without merging**